src/roman_datamodels/stnode/_node.py

```diff
@@ -142,16 +142,19 @@ def __getattr__(self, key):
     def __setattr__(self, key, value):
         """
         Permit assigning dict keys as attributes.
+        If the attribute is assigned a Node subclass (whether correct or not),
+        it will not be validated on assignment.
         """
         if key[0] != "_":
             value = self._convert_to_scalar(key, value)
             if key in self._data:
-                if will_validate():
-                    schema = _get_schema_for_property(self._schema(), key)
-
-                    if schema == {} or _validate(key, value, schema, self.ctx):
-                        self._data[key] = value
-                self.__dict__["_data"][key] = value
+                if not (isinstance(value, DNode) or isinstance(value, LNode)):
+                    if will_validate():
+                        schema = _get_schema_for_property(self._schema(), key)
+                        if schema == {} or _validate(key, value, schema, self.ctx):
+                            # self._data[key] = value
+                            pass
```

Comment:
```suggestion
                        schema == {} or _validate(key, value, schema, self.ctx)
```
The "if statement" serves no purpose. If `schema` is in fact an empty dictionary `_validate` will not be called, otherwise it will.